dom/Dockerfile

```diff
@@ -1,6 +1,24 @@
-FROM buildkite/puppeteer:7.1.0
+FROM alpine:latest
 
 WORKDIR /app
-COPY dom .
```

Comment:
Keep in mind that his `Dockerfile` needs to work for our GitHub Actions :) 
The command used to build the image is [here](https://github.com/01-edu/public/blob/b863e07340cb2af393bd9036a1a338b147b6f56f/.github/workflows/ga-image-build-branch.yml#L45)